terraform/modules/ecs/ecs.tf

```diff
@@ -43,7 +43,7 @@ resource "aws_ecs_service" "this" {
   name            = each.key
   cluster         = aws_ecs_cluster.dibbs_app_cluster.id
   task_definition = each.value.arn
-  desired_count   = local.service_data[each.key].app_count
+  desired_count   = local.service_data[each.key].min_capacity
```

Comment:
Non-blocking...do AWS services support scaling down to 0 with a rapid restart back to 1 if the container is hit? This could save some money, but may have a small performance impact. Just a thought!

Reply:
I'm not sure, but I can look into it. That would be added in a follow-up PR. Changing to min_capacity is only a bit of setup to get us closer to adding autoscaling.

Reply:
I'm not sure if you already created a ticket for the follow-up PR, but there are different methods you can use for scaling. ECS does not natively scaled down to 0 then back up, so you'll have to integrate it with other services. Here's a couple of references that can be added to the follow-up ticket :
- Proactive [method](https://aws.amazon.com/blogs/containers/proactive-scaling-of-amazon-ecs-services-using-amazon-ecs-service-connect-metrics/)
- ECS autoscaling [methods](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/capacity-autoscaling.html)

Reply:
I added a link to these comments to the ticket to do this work so that the context and knowledge are preserved. 💯